js/anxpro.js

```diff
@@ -451,7 +826,99 @@ module.exports = class anxpro extends Exchange {
     }
 
     parseOrder (order, market = undefined) {
-        //
+        if ('orderId' in order)
+            return this.parseV3Order (order, market);
+        else
+            return this.parseV2Order (order, market);
+    }
+
+    parseV3Order (order, market = undefined) {
+        //   { orderType: 'LIMIT',
+        //     tradedCurrency: 'XRP',
+        //     settlementCurrency: 'BTC',
+        //     tradedCurrencyAmount: '400.00000000',
+        //     buyTradedCurrency: true,
+        //     limitPriceInSettlementCurrency: '0.00007129',
+        //     timestamp: '1522547850000',
+        //     orderId: '62a8be4d-73c6-4469-90cd-28b4726effe0',
+        //     tradedCurrencyAmountOutstanding: '0.00000000',
+        //     orderStatus: 'FULL_FILL',
+        //     executedAverageRate: '0.00007127',
+        //     trades:
+        //     [ { tradeId: 'fe16b796-df57-41a2-b6d9-3489f189749e',
+        //         orderId: '62a8be4d-73c6-4469-90cd-28b4726effe0',
+        //         timestamp: '1522547850000',
+        //         tradedCurrencyFillAmount: '107.91298639',
+        //         settlementCurrencyFillAmount: '0.00768772',
+        //         settlementCurrencyFillAmountUnrounded: '0.00768772',
+        //         price: '0.00007124',
+        //         ccyPair: 'XRPBTC' },
+        //         { tradeId: 'e2962f67-c094-4243-8b88-0cdc70a1b1c7',
+        //             orderId: '62a8be4d-73c6-4469-90cd-28b4726effe0',
+        //             timestamp: '1522547851000',
+        //             tradedCurrencyFillAmount: '292.08701361',
+        //             settlementCurrencyFillAmount: '0.02082288',
+        //             settlementCurrencyFillAmountUnrounded: '0.02082288',
+        //             price: '0.00007129',
+        //             ccyPair: 'XRPBTC' } ] }
+        const tradedCurrency = this.safeString (order, 'tradedCurrency');
+        const orderStatus = this.safeString (order, 'orderStatus');
+        let status = undefined;
+        if (orderStatus === 'ACTIVE')
```

Comment:
Same thing as with `parseTransactionStatus`, we're moving to `parseOrderStatus` everywhere: https://github.com/ccxt/ccxt/blob/master/js/kraken.js#L918